app/src/main/java/com/jjswigut/matters/ui/MatterListFragment.kt

```diff
@@ -1,44 +1,44 @@
 package com.jjswigut.matters.ui
 
 import android.os.Bundle
-import androidx.fragment.app.Fragment
-import androidx.recyclerview.widget.GridLayoutManager
-import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.jjswigut.matters.ui.MatterListRecyclerViewAdapter
-import com.jjswigut.matters.R
-import com.jjswigut.matters.dummy.DummyContent
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.jjswigut.matters.databinding.FragmentMatterListBinding
+
 
 /**
  * A fragment representing a list of Items.
  */
 class MatterListFragment : Fragment() {
 
-    private var columnCount = 1
+    private lateinit var binding: FragmentMatterListBinding
+    private lateinit var listAdapter: MatterListRecyclerViewAdapter
 
     override fun onCreateView(
```

Comment:
in general, all onCreateView will and should ever do is create the view.

all of our post view creation logic (like setting up recycler view and what not) should live in `onViewCreated`